spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'PostsController', type: :request do
+  user = User.create(name: 'Tom & Jerry', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Best friends',
+                     PostsCounter: 12)
+  subject do
+    Post.new(title: 'Physics', text: 'This is not my first post', comments_counter: 10, likes_counter: 10,
+             author: user)
+  end
+
+  before { subject.save }
+
+  describe 'GET #index' do
+    before(:each) { get user_posts_path(user_id: 1) }
+    it 'returns http success' do
+      # get '/users/1/posts'
+      # get user_posts_path(1)
```

Comment:
- [ ] Kindly remove commented code as it can make your code look untidy and hard to read